apps/frontend/src/app/services/game.service.ts

```diff
@@ -25,19 +25,23 @@ export class GameService {
    *
    * @returns {Observable<GameDto[]>} - The games
    */
-  public getAllGames(name: string): Observable<GameDto[]> {
+  public getAllGames(name = '', page = 1, pageSize = 20): Observable<GameDto[]> {
+    const params = new HttpParams()
+      .set('name', name)
+      .set('page', page.toString())
+      .set('pageSize', pageSize.toString())
+
     return this.http
-      .get<object[]>(this.gamesUrl, {
-        params: { name },
+      .get<object>(this.gamesUrl, {
```

Comment:
```suggestion
      .get<{ content: object[] }>(this.gamesUrl, {
```
De chames unger im `response?.body as ...` lösche :)

Reply:
Done